echo-events/echo/echo.gradle

```diff
@@ -0,0 +1,66 @@
+
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+  dependencies {
+    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
+    classpath "org.jetbrains.kotlin:kotlin-allopen:$kotlinVersion"
+  }
+}
+
+apply plugin: "io.spinnaker.plugin.service-extension"
+apply plugin: "maven-publish"
+apply plugin: "kotlin"
+apply plugin: "kotlin-kapt"
+apply plugin: "kotlin-spring"
+
+repositories {
+  mavenCentral()
+  jcenter()
+  maven { url "https://spinnaker-releases.bintray.com/jars" }
+}
+
+sourceSets {
+  main {
+    java { srcDirs = ['src/main/java'] }
+  }
+}
+
+spinnakerPlugin {
+  serviceName = "echo"
+  pluginClass = "com.opsmx.plugin.custom.event.CustomEchoPlugin"
+  requires="echo>=0.0.0"
+}
+
+dependencies {
+  compileOnly "org.pf4j:pf4j:"+"${pf4jVersion}"
+  compileOnly "io.spinnaker.echo:echo-api:"+"${echoVersion}"
+  compileOnly (group: 'org.springframework', name: 'spring-context', version: "${springVersion}")
+  compileOnly (group: 'io.spinnaker.kork', name: 'kork-plugins-spring-api', version: "${korkVersion}")
+  compileOnly (group: 'org.springframework', name: 'spring-web', version: "${springVersion}")
+//  compileOnly(platform('io.spinnaker.echo:echo-bom:'+"${echoVersion}"))
+  implementation 'com.fasterxml.jackson.core:jackson-databind:2.0.1'
+  implementation (group: 'org.apache.camel', name: 'camel-rabbitmq', version: "${camelVersion}") {
+    exclude group: 'org.slf4j', module: 'slf4j-api'
+  }
+  implementation (group: 'org.apache.camel', name: 'camel-core', version: "${camelVersion}") {
+    exclude group: 'org.slf4j', module: 'slf4j-api'
+  }
+  implementation (group: 'org.apache.camel', name: 'camel-jackson', version: "${camelVersion}") {
+    exclude group: 'org.slf4j', module: 'slf4j-api'
+  }
+
+}
+
+configurations.all {
```

Comment:
Are you sure about this version? Pls check with the other repos and confirm. And is this Gradle file the top one? if not we might need to move this config to the topmost one.